src/data/communityCurationCohorts.json

```diff
@@ -304,6 +314,11 @@
             "id": 2816,
             "name": "OU",
             "tooltip": "The Open University"
+          },
+          {
+            "id": 3829,
+            "name": "ESCAPE",
+            "tooltip": "ESCAPE"
```

Comment:
Tooltip should be `European Science Cluster of Astronomy & Particle physics ESFRI research infrastructures`

Reply:
I've updated this now, thanks!